package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "napi-build-utils",
+  "version": "1.0.0",
+  "description": "A set of utilities to assist developers of tools building N-API native add-ons",
+  "main": "index.js",
+  "scripts": {
+    "doc": "./node_modules/.bin/jsdoc2md index.js >index.md",
+    "test": "mocha test/ && npm run lint",
+    "lint": "standard",
+    "prepublishOnly": "npm run test && npm run doc"
+
+  },
+  "keywords": [
+    "n-api"
+  ],
+  "author": "Jim Schlight",
+  "license": "MIT",
+  "homepage": "https://github.com/inspiredware/napi-build-utils#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/inspiredware/napi-build-utils.git"
+  },
+  "bugs": {
+    "url": "https://github.com/inspiredware/napi-build-utils/issues"
+  },
+  "devDependencies": {
+    "chai": "^4.1.2",
+    "jsdoc-to-markdown": "^4.0.1",
+    "mocha": "^5.2.0",
+    "standard": "^12.0.1"
+  },
+  "binary": {
+    "for_testing_purposes": "This is not an N-API module.",
```

Comment:
I think that npm will preserve these fields when publishing, but it'd be good to double check just to make sure. That is, publish a version, then `npm install` it and make sure the tests run

Reply:
Good point. I ran `npm run prepublishOnly` and `npm pack`. I checked the contents of the resulting `napi-build-utils-1.0.0.tgz` file and the `binary` property is unscathed.

Reply:
Ahh, excellent. I didn't know about `npm pack`. Very cool!